content/source/docs/enterprise/admin/automated-recovery.html.md

```diff
@@ -87,6 +87,13 @@ apt-get install -y jq
 
 curl https://install.terraform.io/ptfe/stable | bash -s fast-timeouts
 
+# Wait for replicated to start before proceeding
+until replicatedctl system status | grep -q -e '"Replicated": "ready",i' -e '"Retraced": "ready"'; do :
+  sleep 1;
+  echo "Replicated is not yet ready";
+done;
+echo 'Replicated is ready.'
```

Comment:
Since the installation script already uses `jq` before this block of code, it might be better to use `jq` instead of `grep`. That way we're parsing JSON and not lines of text.

```suggestion
until [ "$(replicatedctl system status | jq '.Replicated == "ready" and .Retraced == "ready"')" == "true" ]; do
  sleep 1
  echo "Replicated is not ready."
done
echo "Replicated is ready."
```

Reply:
True.  If someone's having problems running this during a "high stress event" they really should be on the phone with our support anyways :)